### PR TITLE
Documentation: Update README (12.11.2024)

### LINKED ISSUE #110: Documentation: update README
Current README is outdated, does not contain all required information and even has some incorrect statements. Should me updated to the current state of the package.

### PR DESCRIPTION
**Pull request by Gero1999**
## Issue

Closes #110 

## Description

Current README in the main branch is outdated, does not contain all required information and even has some incorrect statements. Should me updated to the current state of the package.

## Definition of Done
Update all relevant content

## How to test
- [x] Installation guidelines work well
- [x] Content information is updated and accurate

`Feel free to modify or add content as well!`

## Contributor checklist
- [ ] Code passes lintr checks
- [ ] Code passes all unit tests
- [ ] New logic covered by unit tests
- [ ] New logic is documented

### CODE REVIEW

**Review by js3110 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,91 +1,66 @@
...
 
-```r
-devtools::install()
-```
+## Contributing 
+### As developer 
+To ensure a clean codebase and smooth cooperation, please adhere to the [contributing guidelines](CONTRIBUTING.md).
 
+### As user 
+Feel free to open identified issues, to reach out to us for questions or fill our [google sheet]() for feedback.
```
Comment (m-kolomanski) [inline README.md:56]: issue: please provide the link to google sheet, if we have any?

File: README.md [hunk 2]
```diff
@@ -1,91 +1,66 @@
...
-<!--
-## Contribute as developer
-
-To ensure a clean and informative git version history, please adhere to the [guidelines](man/GUIDELINES.md) of our git workflow. You can find further information on possible ways to use gits full power on our homemade [cheatsheet](man/GIT-CHEATSHEET.md).
-
--->
-
 <!-- 
 
 ## Getting started
```
Comment (m-kolomanski) [inline README.md:66]: issue: there is a lot of commented out content, please include whatever is relevant and remove the rest.

**Review by m-kolomanski — CHANGES_REQUESTED** [review]
Hi, from my side:

- I do not think it is currently useful to install the package directly from github. We cannot run the application like that since the dependencies will not load, we will need to fix that. So after installation, you still have to change the directory to the installation directory, which makes more sense when the repo is cloned directly - I have left only this part in, as I think it makes the most sense
- I have updated the badges, as the links pointed towards `shiny` repository, not `aNCA`
- I have updated links to contributing guidelines
- I have added a **Quick start* - so the basics of app usage and what the user can do to test it out upon first startup. Please read and verify that it is correct, maybe some other workflow makes more sense - then please change.

In addition, I have some comments and requested changes. The contents seem to be copied over from a template, please make sure only applicable parts are included and that the links point in the right directions.

**Comment by Gero1999** [comment]
> Hi, from my side:
> 
> * I do not think it is currently useful to install the package directly from github. We cannot run the application like that since the dependencies will not load, we will need to fix that. So after installation, you still have to change the directory to the installation directory, which makes more sense when the repo is cloned directly - I have left only this part in, as I think it makes the most sense
> * I have updated the badges, as the links pointed towards `shiny` repository, not `aNCA`
> * I have updated links to contributing guidelines
> * I have added a *_Quick start_ - so the basics of app usage and what the user can do to test it out upon first startup. Please read and verify that it is correct, maybe some other workflow makes more sense - then please change.
> 
> In addition, I have some comments and requested changes. The contents seem to be copied over from a template, please make sure only applicable parts are included and that the links point in the right directions.

Hello Mateusz! You are right, our main branch has commented code that I was not sure if to keep it perhaps for the future, but I just deleted it and solve the issues you mentioned. I will test with some  others non-programming based people to check if they manage to understand completely the instructions on their own, as I know that sometimes this can be a game changer for our users. 

And many thanks for making the changes, I think now it looks much better! 😎

**Review by m-kolomanski — APPROVED** [review]
I agree with @Gotfrid , changing how the package loads dependencies is out of scope of this issue and that is why I have suggested removing the information altogether. Lets create a separate issue as @Gotfrid suggests, and then update README alongside when that option is available.

**Pull request merged**